Header header
geometry_msgs/Vector3 angular_velocity
float64 angular_mag 
geometry_msgs/Vector3 acceleration
float64 acc_mag 
